src/Tribe/Integrations/WPML/Option.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * Handles the translation of the plugin settings.
+ *
+ * @since   TBD
+ *
+ * @package Tribe__Events__Integrations__WPML__Option
+ */
+
+
+/**
+ * Class Tribe__Events__Integrations__WPML__Option
+ *
+ * @since TBD
+ */
+class Tribe__Events__Integrations__WPML__Option {
+
```

Comment:
:no_entry_sign: Class name is not valid; consider Tribe_Events_Integrations_WPML_Option instead

<sub>hash: 4aca20c4324773aedc3206e812d317dd</sub>